.buildbot.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Unless there's a good reason for specifying `bash`, can we use `/bin/sh`? It causes fewer portability problems in the long run.